File: Source/SaveSystem2.0/MAKE REWARDS HERE/SaveObjects/GameAwardsDataSorted.cs
public class GameAwardsDataSorted
{
    public Auras Auras { get; set; }
    public Hats Hats { get; set; }
    public Nitros Nitros { get; set; }
    public Skins Skins { get; set; }
    public Trails Trails { get; set; }
    public Windwalks Windwalks { get; set; }
    public Tournament Tournament { get; set; }
    public Deathless Deathless { get; set; }
    public Wings Wings { get; set; }

    public GameAwardsDataSorted()
    {
        Auras = new Auras();
        Wings = new Wings();
        Deathless = new Deathless();
        Hats = new Hats();
        Nitros = new Nitros();
        Skins = new Skins();
        Trails = new Trails();
        Windwalks = new Windwalks();
        Tournament = new Tournament();
    }
}

File: src/SaveSystem2.0/MAKE REWARDS HERE/SaveObjects/GameAwardsDataSorted.ts
import { Auras } from "./RewardObjects/Auras"
import { Deathless } from "./RewardObjects/Deathless"
import { Hats } from "./RewardObjects/Hats"
import { Nitros } from "./RewardObjects/Nitros"
import { Skins } from "./RewardObjects/Skins"
import { Tournament } from "./RewardObjects/Tournament"
import { Trails } from "./RewardObjects/Trails"
import { Windwalks } from "./RewardObjects/Windwalks"
import { Wings } from "./RewardObjects/Wings"

export class GameAwardsDataSorted {
    public Auras: Auras
    public Hats: Hats
    public Nitros: Nitros
    public Skins: Skins
    public Trails: Trails
    public Windwalks: Windwalks
    public Tournament: Tournament
    public Deathless: Deathless
    public Wings: Wings

    public constructor() {
        this.Auras = new Auras()
        this.Wings = new Wings()
        this.Deathless = new Deathless()
        this.Hats = new Hats()
        this.Nitros = new Nitros()
        this.Skins = new Skins()
        this.Trails = new Trails()
        this.Windwalks = new Windwalks()
        this.Tournament = new Tournament()
    }
}
